boards/hail/src/main.rs

```diff
@@ -393,7 +464,25 @@ pub unsafe fn reset_handler() {
     // Uncomment to measure overheads for TakeCell and MapCell:
     // test_take_map_cell::test_take_map_cell();
 
-    // debug!("Initialization complete. Entering main loop");
+    // Signbus Initialization
```

Comment:
This should all be commented out (but left here) like the `test_take_map_cell` code above